README.md

```diff
@@ -728,6 +728,83 @@ _You can enable the following settings in Xcode by running [this script](resourc
   let dictionary: [String: Any] = [:]
   ```
 
+* <a id='single-line-statement-braces'></a>(<a href='#single-line-statement-braces'>link</a>) The opening brace following a single-line statement should be on the same line as the rest of the statement. [![SwiftFormat: braces](https://img.shields.io/badge/SwiftFormat-braces-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#braces)
+
+  <details>
+
+  ```swift
+  // WRONG
+  if !planet.isHabitable
+  {
+    planet.terraform()
+  }
+
+  class Planet
+  {
+    func terraform()
+    {
+      generateAtmosphere()
+      generateOceans()
+    }
+  }
+
+  // RIGHT
+  if !planet.isHabitable {
+    planet.terraform()
+  }
+
+  class Planet {
+    func terraform() {
+      generateAtmosphere()
+      generateOceans()
+    }
+  }
+  ```
+
+  </details>
+
+* <a id='multi-line-statement-braces'></a>(<a href='#multi-line-statement-braces'>link</a>) The opening brace following a multi-line statement should wrap to a new line. [![SwiftFormat: wrapMultilineStatementBraces](https://img.shields.io/badge/SwiftFormat-wrapMultilineStatementBraces-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#wrapMultilineStatementBraces)
```

Comment:
The style in this rule is already described in other places throughout the style guide (and we already lint for this automatically, since we have the `wrapMultilineStatementBraces` rule enabled). 

It felt like a good idea to add an explicit prose rule for this, to compliment the new rule for the single-statement case.

Reply:
I think that it is reasonable to repeat this a bit in `long-function-declaration`.

Reply:
In `long-function-declaration`, I think this point is covered by:

> Put the open curly brace on the next line so the first executable line doesn't look like it's another parameter.